### PR TITLE
Hotfix/limit person bio

### LINKED ISSUE #626: Limit bio information for non-current candidates
Linked to https://trello.com/c/d1aYISge/87-wcivf-stop-showing-bio-info-for-non-current-candidates

Display a stripped down set of information for a candidate if they do not have a current or future candidacy. For example:

Intro:
'Name' was the 'Party' candidate in 'District' in the 'Election.

Previous elections

### PR DESCRIPTION
**Pull request by michaeljcollinsuk**
Closes #626 

Collaboration with @VirginiaDooley 

- Created a new template for non-current candidates
- Added conditional on PersonView to render appropriate template
- Ensure google does not index non-current candidates

### CODE REVIEW

File: wcivf/apps/people/templates/people/not_current_person_detail.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+{% block content %}
+  <div class="card person_banner">
+    <h2 class="person_name">{{ object.name }}
+    {% if person.death_date and person.current_or_future_candidacies and person.personpost.party.is_independent %}
+      (Deceased)
+    {% endif %}
+    </h2>
+    {% if object.photo_url %}
+      <img src="{{ object.photo_url }}" class="person_photo detail" alt="profile photo of {{ object.name }}">
```
Comment (symroe) [inline wcivf/apps/people/templates/people/not_current_person_detail.html:22]: I don't think we want photos on these pages either!

File: wcivf/apps/people/templates/people/not_current_person_detail.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    {% endif %}
+
+    <p>
+    {{ object.intro|safe }}. Our volunteers have been adding information on {{ object.name }} - here's everything we know so far!
```
Comment (symroe) [inline wcivf/apps/people/templates/people/not_current_person_detail.html:26]: "Our volunteers have been adding information on {{ object.name }} - here's everything we know so far!" can go from this line

File: wcivf/apps/people/templates/people/not_current_person_detail.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      {% for person_post in object.past_not_current_candidacies %}
+      <li><strong>{{ person_post.election.election_date|date:"Y" }}:</strong>
+        <strong>{{ person_post.election }}</strong>, as the {{ person_post.party.party_name }} candidate for
+        <a href="{{ person_post.post_election.get_absolute_url }}">{{ person_post.post.label }}</a>
+        {{ person_post.post_election.short_cancelled_message_html }}
+      {% endfor %}
+      </ul>
+  </div>
+
+  {% if referer_postcode %}
```
Comment (symroe) [inline wcivf/apps/people/templates/people/not_current_person_detail.html:43]: I don't think we're ever going to some from a postcode, so this block can be removed (and the postcode search form added in below)

File: wcivf/apps/people/templates/people/not_current_person_detail.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  {% endif %}
+
+{% if not referer_postcode %}
+  {% include "elections/includes/_postcode_search_form.html" %}
+{% endif %}
+
+{% endblock content %}
+
+{% block breadcrumbs %}
+{% if referer_postcode %}
```
Comment (symroe) [inline wcivf/apps/people/templates/people/not_current_person_detail.html:58]: Same as above

**Comment by symroe** [comment]
This is absolutely the right direction - and will save @pmk01 a lot of emails I think!

Would you mind uploading a screenshot for Peter's :+1: before we merge this?

**Review by symroe — APPROVED** [review]

**Comment by symroe** [comment]
Looks good!

**Comment by VirginiaDooley** [comment]
@pmk01

**Comment by pmk01** [comment]
Will this mean there's no way to get to the relevant candidates.democracyclub.org.uk page from WCIVF? I don't think that's a major issue, but worth thinking about @symroe ?

**Comment by symroe** [comment]
Yeah, there's no link, but I think that's fine - the people coming to Who don't generally click through anyway!

**Pull request merged**